src/contrib/mlir/runtime/cpu/cpu_callbacks.cpp

```diff
@@ -719,7 +719,7 @@ static void __mlir_cblas_sgemm_with_bias(StaticMemRef* memRefmatA,
     }
 }
 
-extern "C" void __mlir_callback_1_input(void* input, void* output, size_t index, OpType type)
+extern "C" void _mlir_ciface_callback_1_input(void* input, void* output, size_t index, OpType type)
```

Comment:
I suggest adding a macro to append the prefix to the func name for better readability. Something like
```
#define MLIR_CALLBACK(NAME) _mlir_ciface_ ## NAME
extern "C" void MLIR_CALLBACK(callback_1_input) ...
```

Reply:
I guess this is more like a personal opinion but I think macros make the code more obscure. I would second this if the macro was replacing more code. However, in this case it's just a prefix with kind of the same number of characters.

Reply:
If the *ciface* naming convention changes, we would have to go and rename all callbacks.